src/VisualStudio/CSharp/Test/CallHierarchy/CSharpCallHierarchyTests.cs

```diff
@@ -474,6 +474,11 @@ class C
         void G$$oo()
         {
         }
+
+        void Foo()
```

Comment:
nit: We avoid Foo these days. Suggest using single letter names such as "M" instead